Sources/RunnerLib/VersionChecker.swift

```diff
@@ -0,0 +1,58 @@
+import DangerShellExecutor
+import Logger
+import Version
+import Foundation
+
+public class VersionChecker {
+    private let shellExecutor: ShellExecutor
+    private let logger: Logger
+
+    public init(
+        shellExecutor: ShellExecutor = .init(),
+        logger: Logger
+    ) {
```

Comment:
Given this is now injectable, can we also add some tests for this please?

Reply:
done!